update website for 1.5.0

roywei opened the pull request:
1. updated installation image
2. updated versions
3. update 1.4.0 to 1.4.1 which should be done in last release. 

@aaronmarkham @IvyBazan @szha @eric-haibin-lin

IvyBazan commented:
Here's a site build preview of the latest changes: 
http://100.24.115.165/

It seems the site nav is still broken.

roywei commented:
Thanks @IvyBazan  It seems the installation is fixed and updated.
This PR is not intended to fix Nav bar and I have no idea how to fix it yet.

IvyBazan commented:
@roywei @aaronmarkham I've updated the preview link to reflect a full site build with 1.5.0 as the new master

roywei commented:
Hi @IvyBazan  are you sure it's updated? 
I have made sure there is not 1.4.0 in the doc any more, removed [all lines](https://github.com/apache/incubator-mxnet/pull/15641/files#diff-9a6e753a2d4579af9a1100510b4fc28bL132) saying:
```
pip install mxnet==1.4.0
```

But somehow it still show up on stage website?

aaronmarkham commented on docs/install/index.md:
```diff
@@ -285,17 +296,24 @@ To build from source, refer to the <a href="ubuntu_setup.html">MXNet Ubuntu inst
 <div class="gpu">
 <div class="pip">
-<div class="v1-4-0">
+<div class="v1-5-0">
```
This is the second mention of v1-5-0 and will mean more stuff will appear.

aaronmarkham commented on docs/install/index.md:
```diff
@@ -285,17 +296,24 @@ To build from source, refer to the <a href="ubuntu_setup.html">MXNet Ubuntu inst
 
-</div> <!-- End of v1-4-0 -->
+</div> <!-- End of v1-5-0 -->
+<div class="v1-4-1">
```
Second mention of v1-4-1 ...

aaronmarkham commented on docs/install/index.md:
````diff
@@ -599,12 +617,18 @@ Refer to the <a href="ubuntu_setup.html#install-the-mxnet-package-for-perl">Perl
 ```
-</div> <!-- End of v1-4-0 -->
+</div> <!-- End of v1-5-0 -->
+<div class="v1-4-1">
````
Third mention...

aaronmarkham commented on docs/install/index.md:
```diff
@@ -892,13 +916,20 @@ For more installation options, refer to the <a href="osx_setup.html">MXNet macOS
 
-</div> <!-- End of v1-4-0 -->
+</div> <!-- End of v1-5-0 -->
+<div class="v1-4-1">
```
4th mention...

roywei commented:
I still don't understand how multiple mentions of versions in <div> will break things. I checked all <div> has propper closing </div>.  Although the parsing may not be robust, but the problem is `pip install mxnet==1.4.0` is showing up even if I removed all appearance of such text.  
Since such text only appears and mess up installation page only when build all versions (v1.5.0 stand alone is fine), the only reason could be website code is wrong in v1.4.0 and the text is checked out from here: https://github.com/apache/incubator-mxnet/blob/master/docs/build_version_doc/build_all_version.sh#L159

Given that, I think it will take longer time for me to figer out the root cause of the bug. I'd like to separate address this issue. I have removed related item in description.

abhinavs95 commented:
@mxnet-label-bot add [Website]

ChaiBapchya commented:
the reason why
```
$ pip install mxnet==1.4.0
```
continues to exist is because for some weird reason
only that particular div class = “v1-4-0” doesn’t have style=“display: none;” 

![Screen Shot 2019-07-26 at 1 09 22 PM](https://user-images.githubusercontent.com/10992635/61978656-b2296500-afa6-11e9-856c-961cb671c61f.png)

If somehow, we are able to set the CSS using JS, we can get rid of this 1.4.0 issue.

I tried to see the onclick event for buttons in the version related dropdown. but it looks generic and hence can’t seem to single out why only v1.4.0 div isn’t getting CSS.

zhreshold commented:
I've double checked http://100.24.115.165/install/index.html?platform=Linux&language=Python&processor=CPU there's no 1.4.0 anymore, am I missing something?

ChaiBapchya commented:
Yes. As mentioned in other duplicate PR #15673 
For version 1.5.0
http://100.24.115.165/versions/1.5.0/install/index.html?platform=Linux&language=Python&processor=CPU

aaronmarkham commented:
You still need settings.ini updated but let's do that in a follow-up. These changes are good.
